webpack/scenes/Subscriptions/SubscriptionsPage.js

```diff
@@ -38,6 +34,48 @@ class SubscriptionsPage extends Component {
       },
     });
 
+    const onPaginationChange = (pagination) => {
+      this.props.loadSubscriptions({
+        ...pagination,
+      });
+    };
+
+    if (subscriptions.results.length === 0 && subscriptions.searchIsActive) {
+      return (
+        <p>{__('No subscriptions match your search criteria.')}</p>
```

Comment:
This isn't really consistent with how we are displaying these messages on other pages and with how patternfly displays the same.

![screen shot 2018-04-04 at 10 47 12 am](https://user-images.githubusercontent.com/4116405/38315036-99ec579c-37f5-11e8-9c3f-b6a06c87bc5e.png)
![screen shot 2018-04-04 at 10 46 45 am](https://user-images.githubusercontent.com/4116405/38315038-99fabd8c-37f5-11e8-95b0-bf1d817383dd.png)


See https://rawgit.com/patternfly/patternfly/master-dist/dist/tests/table-view-navbar.html#

Reply:
I see, thanks. I'll update it.